tests/integration_tests/test_gym_envs.py

```diff
@@ -0,0 +1,130 @@
+from itertools import product
+from multiprocessing import Process
+import pickle
+import unittest
+
+import gym
+from nose2.tools import params
+
+from garage.tf.envs.base import TfEnv
+from garage.theano.envs.base import TheanoEnv
+
+# White list of environments that cannot be rendered
```

Comment:
Please see the test suite I implemented in branch `env_pickle_tests`. I think you should be able to commit all of it along with this PR.